src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -0,0 +1,196 @@
+
+Portals is a new HTML element that can help you keep your front-end simple while allowing seamless navigations with custom transitions. The best part is that Portals is quite easy to work with. Think of it as an iframe in that it allows for embedding, but unlike an iframe it also comes with a feature to directly navigate into its content. Seeing is believing: please first check out what we showcased in Chrome Dev Summit last year.
+
+<div style="width:100%; padding-top: 56.25%; position: relative;">
+  <iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://www.youtube.com/embed/Ai4aZ9Jbsys?start=1081" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+
+As you saw in the manga viewer demo, the current experience was based on classic navigations where the reader had to wait with a blank screen until the browser finished to render the next chapter. Whereas the improved experience with Portals was pre-rendering the next chapter, triggered an animation on user interaction, and finally activated the portal to create a magical seamless experience. The demo didn’t show it (because we used Chrome DevTools mobile simulator 😛) but when the pre-rendered chapter on the bottom left corner fully animated and covered the screen, the URL also switched to the next chapter.
+
+Before Portals, we could have rendered another page using an iframe. We could also have added animations to move the frame around the page. But iframes won’t let you navigate into their content. Portals close this gap, enabling interesting use cases.
+
+# Try out Portals in Chrome Canary
+
+You can try out Portals in Chrome Canary by flipping an experimental flag (chrome://flags/#enable-portals). Once Portals is enabled, confirm in DevTools that you have the new shiny `HTMLPortalElement`.
+
+![A screenshot of the DevTools console showing the HTMLPortalElement](HTMLPortalElement.png)
+
+Let’s walk through a basic example.
+
+```javascript
+// Create a portal with the wikipedia page, and embed it
+// (like an iframe). You can also use <portal> tag instead.
+portal = document.createElement('portal');
+portal.src = 'https://en.wikipedia.org/wiki/World_Wide_Web';
+portal.style = '...';
+document.body.appendChild(portal);
+
+// When the user touches the preview (embedded portal):
+// do fancy animation, e.g. expand …
+// and finish by doing the actual transition
+portal.activate();
+```
+
+It’s that simple. Try this code in the DevTools console, the wikipedia page should open up. If you wanted to build something like we showed in CDS, the following snippet will be of interest.
+
+![A gif of preview portal style demo](portal-preview-demo.gif)
+
+```javascript
```

Comment:
Instead of this snippet, would you be ok turning this into a glitch using https://glitch.com/ ?

Then we can embed a running example on the site. You could also include a running version of the previous wikipedia example.

Reply:
If so, can I 
1. Keep this snippet as is (as this is just for reference code)
2. Remove the GIF below and add the running glitch (https://uskay-portals-demo.glitch.me)

Reply:
Rob, I was considering how I should be using glitch embed but I think I'll stick with not using glitch embeds this time. The reason is: Portals can only be used in Chrome Canary now with an experimental flag on. I suppose most of the people will be reading this on unsupported browsers which doesn't really make sense to have the demos/snippets embedded as glitch (it will not run or show fallback stuff which is not ideal).